functions/display-logic.php

```diff
@@ -214,18 +139,105 @@ function display_front_page_issue_details() {
 
 	ob_start();
 ?>
-	<a class="fp-issue-link" href="<?php echo get_permalink( $current_issue->ID ); ?>">
-		<h2 class="h3 fp-subheading fp-issue-title">In This Issue</h2>
+	<div class="fp-issue position-relative mb-4 mb-md-0 hover-parent">
+		<h2 class="fp-issue-title text-uppercase my-3">In This Issue</h2>
 
 		<?php if ( $current_issue_thumbnail ): ?>
-		<img class="img-responsive center-block fp-issue-img" src="<?php echo $current_issue_thumbnail; ?>" alt="<?php echo $current_issue_title; ?>" title="<?php echo $current_issue_title; ?>">
+		<img class="img-fluid hover-child-filter-brightness" src="<?php echo $current_issue_thumbnail; ?>" alt="<?php echo $current_issue_title; ?> Cover Image">
```

Comment:
alt text can be made empty here, since the remaining link text already provides the same descriptive text